.github/workflows/build-windows.yml

```
@@ -50,31 +50,31 @@ jobs:
       - name: Upload Artifact
         uses: actions/upload-artifact@v4
         with:
-          name: DailyReminder-Windows-x64
-          path: DailyReminder-Windows-x64.zip
+          name: TrainSimulationApp-Windows-x64
+          path: TrainSimulationApp-Windows-x64.zip
           retention-days: 30
 
       - name: Create Release
         if: startsWith(github.ref, 'refs/tags/v')
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)